pkg/k8shandler/desirednodestate.go

```diff
@@ -165,6 +165,7 @@ func (cfg *desiredNodeState) getLabels() map[string]string {
 	labels["es-node-data"] = strconv.FormatBool(cfg.isNodeData())
 	labels["es-node-master"] = strconv.FormatBool(cfg.isNodeMaster())
 	labels["cluster-name"] = cfg.ClusterName
+	labels["component"] = cfg.ClusterName
```

Comment:
Should this be `es` which I think is consistent with what we add to the pod spec

Reply:
Can be either:
* `elasticsearch` - ops and app in one cluster
* `elasticsearch-apps` - app cluster
* `elasticsearch-infra` - ops cluster

https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/logstore.go#L103-L117